aten/src/ATen/native/Convolution.cpp

```diff
@@ -30,6 +31,7 @@ struct ConvParams {
   bool is_padding_neg() const;
   bool is_stride_neg() const;
   void view1d_as_2d();
+  bool use_caffe2_depthwise(const at::Tensor& input, const at::Tensor& weight) const;
```

Comment:
I don't know if we should call it "caffe2" or something closer to its intrinsic characteristic like "use_depthwise3x3_winograd"? or somewhere in-between like "use_depthwise_cpu"?